source/pages/Send/SendSys.tsx

```diff
@@ -60,8 +60,18 @@ export const SendSys = () => {
     ? Object.values(activeAccount.assets.syscoin)
     : [];
 
+  const formattedAssetBalance =
+    selectedAsset &&
+    truncate(
+      formatCurrency(
+        String(+selectedAsset.balance / 10 ** selectedAsset.decimals),
```

Comment:
The fix seems to work well, but I noticed, for now, we have any type of syscoin assets. It's best to ensure we always will have decimals field for syscoin assets.
https://github.com/syscoin/pali-wallet/blob/ba7eb368b2a102793e1f86acc1768a3ec7f808be/source/state/vault/types.ts#L49